CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Welcome to Ceph NVMe-oF Gateway contributing guide <!-- omit in toc -->
+
+## New contributor guide
+
+The Ceph NVMe-oF Gateway project pivots around 2 other major Open Source projects:
+- [SPDK](https://spdk.io/) [[source code](https://github.com/spdk/spdk/)], whih internally relies on the DPDK project.
+- [Ceph](https://ceph.com/) [[source code](https://github.com/ceph/ceph)].
+
+## Reporting issues
+
+If you found an issue, please identify first whether the issue comes from the NVMe-oF Gateway, or any of the underlying components:
+* For NVMe-oF Gateway issues (usually a Python traceback), please check if [the issue has already been reported](https://github.com/ceph/ceph-nvmeof/issues).
+  If not, please [open a new one](https://github.com/ceph/ceph-nvmeof/issues/new).
+* For SPDK-related issues, please [open an issue in the SPDK Github Issues](https://github.com/spdk/spdk/issues/).
+* For Ceph-related issues, please [open an issue in the Ceph issue tracker](https://tracker.ceph.com/).
+
+## Submitting changes
+
+### Coding conventions
+
+This project follows:
+* Python coding guidelines ([PEP-8](https://peps.python.org/pep-0008/)).
+* [gRPC and Protocol Buffers](https://grpc.io/docs/what-is-grpc/introduction/).
+
+### Commit format
+
+When committing your changes, please:
+* Sign-off (`git commit -s`), which will automatically add the following trailer to your commit: `Signed-off-by: FirstName LastName <email@example.com>`.
+  This constitues your [Developer Certificate of Origin](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin), and is enforced by a [CI check](https://probot.github.io/apps/dco/). 
+* Follow the [Conventional Commit syntax](https://www.conventionalcommits.org/en/v1.0.0/). This is not yet enforce via CI checks.
+
+### Testing
+
+TODO.
+
+### Pull request
+
+Refer to Ceph's ["Submitting Patches to Ceph"](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst) documentation, with the difference that:
+* This project uses Github Issues instead of the Ceph Tracker,
+* Therefore commits and pull request should use the `Fixes #12345` syntax (where `#12345` is the Gituub issue number), instead of the `Fixes: https://tracker.ceph.com...`.
```

Comment:
```suggestion
* Therefore commits and pull requests should use the `Fixes: #12345` syntax (where `#12345` is the Github issue number), instead of the `Fixes: https://tracker.ceph.com/...`.
```